solana/ts/lib/utils.ts

```diff
@@ -159,3 +97,15 @@ export const quoterAddresses = (programId: PublicKeyInitData) => {
     registeredNttAccount,
   };
 };
+
+//   // The `translateError` function expects this format, but the idl gives us a
+//   // different one, so we preprocess the idl and store the expected format.
+//   // NOTE: I'm sure there's a function within anchor that does this, but I
+//   // couldn't find it.
+//   private processErrors(): Map<number, string> {
```

Comment:
should we delete this?

Reply:
ah yes, not sure if you wrote the note initially but I did find the correct method to use for processing the errors 

https://github.com/wormhole-foundation/example-native-token-transfers/blob/sdk/solana-dedupe/solana/ts/lib/ntt.ts#L199-L206